src/backend/distributed/executor/multi_task_tracker_executor.c

```diff
@@ -879,13 +881,14 @@ TrackerConnectPoll(TaskTracker *taskTracker)
 			if (pollStatus == CLIENT_CONNECTION_BUSY_READ ||
 				pollStatus == CLIENT_CONNECTION_BUSY_WRITE)
 			{
-				uint32 maxCount = REMOTE_NODE_CONNECT_TIMEOUT / RemoteTaskCheckInterval;
```

Comment:
I think `ceil(NodeConnectionTimeout * 1.0f / RemoteTaskCheckInterval)` is always > 0, provided that `NodeConnectionTimeout > 0`